ksql-engine/src/main/java/io/confluent/ksql/function/udf/url/UrlParser.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ **/
+
+package io.confluent.ksql.function.udf.url;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+
+public abstract class UrlParser {
```

Comment:
Covert to a utility class?

Reply:
Utility class:
 - static methods
 - `final class`
 - private constructor.

:)